lib/tasks/testSetup.js

```diff
@@ -167,7 +167,11 @@ function waitClosed(done) {
    var response = shell.exec(`docker stack ls | grep ${Options.stack}`, {
       silent: true,
    });
-   if (response.stdout != "") {
+   //Need to wait for the network to be removed or waitBoot will fail.
+   var network = shell.exec(`docker network ls | grep ${Options.stack}`, {
```

Comment:
I noticed in the other PR that we used: ``` shell.exec().grep()``` for windows compatibility.  How about here?

Reply:
Sorry missed a commit